src/app/cli/src/init/test_block_command_verification.ml

```diff
@@ -0,0 +1,123 @@
+open Core_kernel
+open Async_kernel
+
+let time_block_command_verification ~(large_precomputed_json_file : string)
+    ~(log_directory : string) =
+  let signature_kind = Mina_signature_kind.t_DEPRECATED in
```

Comment:
Since this is a newly added command, I don't see a problem making this configurable by CLI input.

Reply:
I tried that originally, but setting the signature kind to anything other than the compiled-config default didn't seem to work, I'm assuming because the signature kind refactor hasn't progressed far enough. It feels a little misleading to expose it as a command-line flag if it doesn't actually work yet.